charts/renku/values.yaml

```diff
@@ -580,26 +580,37 @@ gateway:
   ingress:
     enabled: false
 
-## Configuration for renku-graph - disabled by default until renku 0.4.0
+## Configuration for renku-graph
 graph:
+  ## Enable/disable the graph - disabled by default until renku 0.4.0
   enabled: false
-#   jena:
-#     admin:
-#       password:
-
-#   gitlab:
-#     url:
-
-#   webhookService:
-#     play:
-#       secret:
-#     hookToken:
-#       secret:
-
-#   triplesGenerator:
-#     play:
-#       secret:
-
-#  tokenRepository:
-#    tokenEncryption:
-#      secret:
+  ## Configuration for Apache Jena
```

Comment:
@rokroskar why do we need to define these values at all? I thought that once `graph` is disabled we don't have to have any values for it and thus has the yaml files a bit less noisy. Unfortunately, there's this one mandatory postgres password value in the `global` section but my thinking was that's the only one.

Reply:
Because it's defined as a dependency in `requirements.yaml` the chart gets pulled and placed in the charts directory. It's rendered with all the rest of the charts, it's just not deployed. So it fails in the rendering step if the values are missing.

Reply:
We should put a conditional in the `secrets.yaml` definitions in the graph chart to skip them if `enable: false` is set. So this here is a place-holder solution without having to change the graph chart for the moment.

Reply:
Right, in that case, we have no choice. It's a bit unfortunate, though.

Reply:
I agree -- lets do it this way for now and when we iterate on the graph chart add in the conditional. Could you make an issue for it?